src/signin.js

```diff
@@ -0,0 +1,97 @@
+function togglePasswordVisibility() {
+  const passwordField = document.getElementById("password");
+  const eye = document.querySelector('.eye')
+  if (passwordField.type === "password") {
+    passwordField.type = "text";
+    eye.classList.toggle('toggle-password');
+    eye.classList.toggle('toggle-password-pressed');
+  } else {
+    passwordField.type = "password";
+    eye.classList.toggle('toggle-password');
+    eye.classList.toggle('toggle-password-pressed');
+  }
```

Comment:
그냥 하나의 클래스만 토글하는 정도면 toggle 메서드를 사용하는 게 편리하겠지만 input type까지 직접 넣어주는 상황에서는 해당 input에 맞는 classname을 정확하게 넣어주는 게 좋을 듯 합니다. 정합성을 위해서요!

```suggestion
  if (passwordField.type === "password") {
    passwordField.type = "text";
    eye.classList.add('toggle-password');
    eye.classList.remove('toggle-password-pressed');
  } else {
    passwordField.type = "password";
    eye.classList.remove('toggle-password');
    eye.classList.add('toggle-password-pressed');
  }
```